[2023-06-14T09:17:14.580+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T09:17:14.589+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T09:17:14.590+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T09:17:14.590+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-06-14T09:17:14.591+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T09:17:14.599+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-06-13 00:00:00+00:00
[2023-06-14T09:17:14.604+0000] {standard_task_runner.py:55} INFO - Started process 54 to run task
[2023-06-14T09:17:14.607+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_postgres', 'create_table', 'scheduled__2023-06-13T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/postgress_db.py', '--cfg-path', '/tmp/tmpl4r62mrd']
[2023-06-14T09:17:14.608+0000] {standard_task_runner.py:83} INFO - Job 27: Subtask create_table
[2023-06-14T09:17:14.675+0000] {task_command.py:388} INFO - Running <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [running]> on host 54f992379a81
[2023-06-14T09:17:14.725+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_postgres
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-06-14T09:17:14.726+0000] {sql.py:254} INFO - Executing: drop table if exists users;
            CREATE TABLE IF NOT EXISTS USERS (
                id INT not null unique,
                name VARCHAR(255) NOT NULL,
                email VARCHAR(255) NOT NULL,
                phone VARCHAR(255) NOT NULL,
                address VARCHAR(255) NOT NULL,
                organization VARCHAR(255) NOT NULL
            );
[2023-06-14T09:17:14.734+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T09:17:14.844+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T09:17:14.849+0000] {sql.py:375} INFO - Running statement: drop table if exists users;
            CREATE TABLE IF NOT EXISTS USERS (
                id INT not null unique,
                name VARCHAR(255) NOT NULL,
                email VARCHAR(255) NOT NULL,
                phone VARCHAR(255) NOT NULL,
                address VARCHAR(255) NOT NULL,
                organization VARCHAR(255) NOT NULL
            );, parameters: None
[2023-06-14T09:17:14.875+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_postgres, task_id=create_table, execution_date=20230613T000000, start_date=20230614T091714, end_date=20230614T091714
[2023-06-14T09:17:14.899+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-06-14T09:17:14.917+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-14T13:01:50.601+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T13:01:50.616+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T13:01:50.617+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T13:01:50.618+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-06-14T13:01:50.618+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T13:01:50.637+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-06-13 00:00:00+00:00
[2023-06-14T13:01:50.645+0000] {standard_task_runner.py:55} INFO - Started process 139 to run task
[2023-06-14T13:01:50.650+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_postgres', 'create_table', 'scheduled__2023-06-13T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/postgress_db.py', '--cfg-path', '/tmp/tmpwbqji7z2']
[2023-06-14T13:01:50.651+0000] {standard_task_runner.py:83} INFO - Job 5: Subtask create_table
[2023-06-14T13:01:50.721+0000] {task_command.py:388} INFO - Running <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [running]> on host 73219206ba7b
[2023-06-14T13:01:50.776+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_postgres
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-06-14T13:01:50.777+0000] {sql.py:254} INFO - Executing: CREATE TABLE IF NOT EXISTS my_table (
  id INT UNIQUE NOT NULL,
  current_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);
[2023-06-14T13:01:50.796+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 255, in execute
    hook = self.get_db_hook()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 179, in get_db_hook
    return self._hook
  File "/home/airflow/.local/lib/python3.7/site-packages/cached_property.py", line 36, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 141, in _hook
    conn = BaseHook.get_connection(self.conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/connection.py", line 435, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `postgres` isn't defined
[2023-06-14T13:01:50.799+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=database_postgres, task_id=create_table, execution_date=20230613T000000, start_date=20230614T130150, end_date=20230614T130150
[2023-06-14T13:01:50.807+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 5 for task create_table (The conn_id `postgres` isn't defined; 139)
[2023-06-14T13:01:50.824+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-06-14T13:01:50.847+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-06-14T14:05:45.034+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T14:05:45.041+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T14:05:45.041+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T14:05:45.042+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-06-14T14:05:45.042+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T14:05:45.050+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-06-13 00:00:00+00:00
[2023-06-14T14:05:45.055+0000] {standard_task_runner.py:55} INFO - Started process 77 to run task
[2023-06-14T14:05:45.058+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_postgres', 'create_table', 'scheduled__2023-06-13T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/postgress_db.py', '--cfg-path', '/tmp/tmpt8bc42h6']
[2023-06-14T14:05:45.059+0000] {standard_task_runner.py:83} INFO - Job 39: Subtask create_table
[2023-06-14T14:05:45.110+0000] {task_command.py:388} INFO - Running <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [running]> on host 5fd7c6d884b5
[2023-06-14T14:05:45.159+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_postgres
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-06-14T14:05:45.161+0000] {sql.py:254} INFO - Executing: CREATE TABLE IF NOT EXISTS my_table (
  id INT UNIQUE NOT NULL,
  execution_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);
[2023-06-14T14:05:45.169+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T14:05:45.287+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T14:05:45.295+0000] {sql.py:375} INFO - Running statement: CREATE TABLE IF NOT EXISTS my_table (
  id INT UNIQUE NOT NULL,
  execution_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);, parameters: None
[2023-06-14T14:05:45.304+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_postgres, task_id=create_table, execution_date=20230613T000000, start_date=20230614T140545, end_date=20230614T140545
[2023-06-14T14:05:45.356+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-06-14T14:05:45.372+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-14T15:24:23.233+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T15:24:23.247+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T15:24:23.248+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T15:24:23.263+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-06-14T15:24:23.264+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T15:24:23.297+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-06-13 00:00:00+00:00
[2023-06-14T15:24:23.310+0000] {standard_task_runner.py:55} INFO - Started process 122 to run task
[2023-06-14T15:24:23.313+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_postgres', 'create_table', 'scheduled__2023-06-13T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/postgress_db.py', '--cfg-path', '/tmp/tmpobwozfgc']
[2023-06-14T15:24:23.327+0000] {standard_task_runner.py:83} INFO - Job 55: Subtask create_table
[2023-06-14T15:24:23.421+0000] {task_command.py:388} INFO - Running <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [running]> on host 3de891eadda2
[2023-06-14T15:24:23.503+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_postgres
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-06-14T15:24:23.506+0000] {sql.py:254} INFO - Executing: CREATE TABLE IF NOT EXISTS execution_table (
  id INT UNIQUE NOT NULL,
  exectime TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);
[2023-06-14T15:24:23.516+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T15:24:23.683+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T15:24:23.689+0000] {sql.py:375} INFO - Running statement: CREATE TABLE IF NOT EXISTS execution_table (
  id INT UNIQUE NOT NULL,
  exectime TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);, parameters: None
[2023-06-14T15:24:23.719+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_postgres, task_id=create_table, execution_date=20230613T000000, start_date=20230614T152423, end_date=20230614T152423
[2023-06-14T15:24:23.777+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-06-14T15:24:23.835+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-14T15:31:56.399+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T15:31:56.406+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-06-14T15:31:56.407+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T15:31:56.407+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-06-14T15:31:56.407+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T15:31:56.425+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-06-13 00:00:00+00:00
[2023-06-14T15:31:56.432+0000] {standard_task_runner.py:55} INFO - Started process 90 to run task
[2023-06-14T15:31:56.436+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'database_postgres', 'create_table', 'scheduled__2023-06-13T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/postgress_db.py', '--cfg-path', '/tmp/tmp60u3ffp5']
[2023-06-14T15:31:56.437+0000] {standard_task_runner.py:83} INFO - Job 63: Subtask create_table
[2023-06-14T15:31:56.487+0000] {task_command.py:388} INFO - Running <TaskInstance: database_postgres.create_table scheduled__2023-06-13T00:00:00+00:00 [running]> on host 9268ba351b9c
[2023-06-14T15:31:56.539+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=database_postgres
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-06-14T15:31:56.541+0000] {sql.py:254} INFO - Executing: drop table if exists execution_table;
CREATE TABLE IF NOT EXISTS execution_table (
  id SERIAL PRIMARY KEY,
  exectime TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);
[2023-06-14T15:31:56.549+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T15:31:56.661+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-06-14T15:31:56.666+0000] {sql.py:375} INFO - Running statement: drop table if exists execution_table;
CREATE TABLE IF NOT EXISTS execution_table (
  id SERIAL PRIMARY KEY,
  exectime TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);, parameters: None
[2023-06-14T15:31:56.686+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=database_postgres, task_id=create_table, execution_date=20230613T000000, start_date=20230614T153156, end_date=20230614T153156
[2023-06-14T15:31:56.729+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-06-14T15:31:56.749+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
